package.json

```diff
@@ -76,6 +76,7 @@
     "react-dom": "^18.3.1",
     "react-outside-click-handler": "^1.3.0",
     "sharp": "^0.33.5",
+    "usehooks-ts": "^3.1.0",
```

Comment:
question: Used anywhere?

Reply:
Tried using it for the on-hover routes, but it was flaky even with this hook. Will remove it if I make it work on-hover in some way. Thinking of using the ::after's 'content' rn.